packages/datatrak-web-server/src/app/createApp.ts

```diff
@@ -71,8 +71,29 @@ export function createApp() {
     .get<SingleSurveyResponseRequest>('surveyResponse/:id', handleWith(SingleSurveyResponseRoute))
     .use('signup', forwardRequest(WEB_CONFIG_API_URL, { authHandlerProvider }))
     // Forward everything else to central server
-    .use('*', forwardRequest(CENTRAL_API_URL, { authHandlerProvider }))
-    .build();
+    .use('*', forwardRequest(CENTRAL_API_URL, { authHandlerProvider }));
+
+  await builder.initialiseApiClient([
+    { entityCode: 'DL', permissionGroupName: 'Public' }, //	Demo Land
+    { entityCode: 'FJ', permissionGroupName: 'Public' }, //	Fiji
+    { entityCode: 'CK', permissionGroupName: 'Public' }, //	Cook Islands
+    { entityCode: 'PG', permissionGroupName: 'Public' }, //	Papua New Guinea
+    { entityCode: 'SB', permissionGroupName: 'Public' }, //	Solomon Islands
+    { entityCode: 'TK', permissionGroupName: 'Public' }, //	Tokelau
+    { entityCode: 'VE', permissionGroupName: 'Public' }, //	Venezuela
+    { entityCode: 'WS', permissionGroupName: 'Public' }, //	Samoa
+    { entityCode: 'KI', permissionGroupName: 'Public' }, //	Kiribati
+    { entityCode: 'TO', permissionGroupName: 'Public' }, //	Tonga
+    { entityCode: 'NG', permissionGroupName: 'Public' }, //	Nigeria
+    { entityCode: 'VU', permissionGroupName: 'Public' }, //	Vanuatu
+    { entityCode: 'AU', permissionGroupName: 'Public' }, //	Australia
+    { entityCode: 'PW', permissionGroupName: 'Public' }, //	Palau
+    { entityCode: 'TL', permissionGroupName: 'Public' }, //	Timor-Leste
+    { entityCode: 'NU', permissionGroupName: 'Public' }, //	Niue
+    { entityCode: 'TV', permissionGroupName: 'Public' }, //	Tuvalu
```

Comment:
This list is copied across and used in a few different places (tupaia-web-server, web-config-server, datatrak-web-server). I'm inclined to keep it as is because:
1. I like having something like a public access list be explicit
2. It's possible that we'd like these apps to deviate (not so much tuapia-web and web-config-server, but datatrak-web it does make sense)
3. There doesn't seem to be a comfortable place to put this (perhaps in `utils` we coud add a `constants` module?)